save-cloud-common/build.gradle.kts

```diff
@@ -45,6 +45,12 @@ kotlin {
                 implementation(libs.spring.security.core)
                 implementation(libs.jackson.module.kotlin)
                 implementation(libs.hibernate.jpa21.api)
+                api(libs.slf4j.api)
+            }
+        }
+        val jvmTest by getting {
+            dependencies {
+                implementation(libs.kotlin.test)
```

Comment:
To execute tests on JVM you should also configure test runner, e.g. `useJunitPlatform`. Currently those tests are probably not running

Reply:
looks like it's not required to specify explicitly junit platform.

tested by run `save-cloud-common:check` with the following changes
```
diff --git a/save-cloud-common/build.gradle.kts b/save-cloud-common/build.gradle.kts
index fc710ba6..59784c15 100644                                                     
--- a/save-cloud-common/build.gradle.kts                                            
+++ b/save-cloud-common/build.gradle.kts                                            
@@ -52,6 +52,9 @@ kotlin {                                                          
             dependencies {                                                         
                 implementation(libs.kotlin.test)                                   
             }                                                                      
+//            tasks.withType<Test> {                                               
+//                useJUnitPlatform()
+//            }
         }
     }
 }
diff --git a/save-cloud-common/src/jvmTest/kotlin/com/saveourtool/save/entities/ExecutionTest.kt b/save-cloud-common/src/jvmTest/kotlin/com/saveourtool/save/entities/ExecutionTest.kt
index 69e4f0fa..27abaa58 100644
--- a/save-cloud-common/src/jvmTest/kotlin/com/saveourtool/save/entities/ExecutionTest.kt
+++ b/save-cloud-common/src/jvmTest/kotlin/com/saveourtool/save/entities/ExecutionTest.kt
@@ -25,7 +25,7 @@ internal class ExecutionTest {
         execution.testSuiteIds = null
         assertNull(execution.parseAndGetTestSuiteIds())

-        execution.testSuiteIds = "1, 2, 3"
+        execution.testSuiteIds = "1, 2, 3, 4"
         assertNotNull(execution.parseAndGetTestSuiteIds()) {
             assertEquals(listOf(1L, 2L, 3L), it)
         }
```

Reply:
but according to [docs](https://kotlinlang.org/docs/gradle.html#set-dependencies-on-test-libraries), need to add platform specific dependency: `kotlin-test-junit`.
Checking